docs/connector-development/connector-builder-ui/maintenance.md

```diff
@@ -0,0 +1,175 @@
+* Configure what we want to be validated: Using `acceptance-test-config.yml`, you can toggle some of the validations that are performed. The most common cases are:
+    * If `expected_records.jsonl` to validate the records that the source returns has been added, ensure `acceptance_tests.basic_read.tests.expect_records` is populated. This might require you to create a catalog for this test suites.
+    * If a stream was made incremental, ensure `acceptance_tests.incremental.tests.expect_records` is populated. This might require you to create a catalog for this test suites.
+* Set up the configuration: In order to run properly, CATs require a valid config. Those configs often rely on secret. You can use [GSM](https://console.cloud.google.com/security/secret-manager?project=dataline-integration-testing) to modify the existing configuration. Simply write the connector name in the filter bar, click on `SECRET_SOURCE-<name>__CREDS` and `NEW VERSION` and upload the new config.
+
+To execute CATs, there are two options.
+
+### Locally
+We recommend using `cd airbyte-integrations/connections/source-<name> && ./acceptance-test-docker.sh` to run them using docker. You might have to change the permissions to run this file. Once it's done, the results will be printed to the terminal.
+
+### In a GitHub pull request
+Given that you have pushed you changes on a branch in airbytehq/airbyte, go on https://github.com/airbytehq/airbyte and you should see a yellow banner saying "<branch> had recent pushes less than a minute ago". Click on "Compare & pull request", fill out the template and click on "Create pull request". A check named "Connectors tests / Connectors CI" will run. Once it's done, a new comment will be added to show the test results. 
+
+## Release
+
+Goal: Make the connector available to end-users
+
+Expected output: https://hub.docker.com/r/airbyte/source-<name>/tags shows a recently created tag
+
+The following steps assume that you are getting
+
+* Bump version (Docker file and metadata.yaml) and update changelogs under `docs/integrations`
+* Fill the Connector Checklist in the pull request
+* If there are breaking changes, follow the process defined by [Connector Breaking Change Release Playbook](https://docs.google.com/document/d/1VYQggHbL_PN0dDDu7rCyzBLGRtX-R3cpwXaY8QxEgzw/edit#heading=h.r2xzw0kclodc). A breaking change is a change that requires the user to take action. It can be triggered by a change in the schema, in the config or in the connector behavior. 
+* In some specific oauth cases, there are platform changes to be done. <expand on this eventually> 
```

Comment:
maybe the step here is just to make sure there's a ticket with the appropriate priority?

If it's a new oauth connector, the platform changes are required before publishing it to cloud. Otherwise they should only be needed if there's a change in the scope requirements